### ISSUE TITLE
Messages of the RSS bot don't get bridged to telegram

### ISSUE THREAD

**Issue by maeries** [mautrix/telegram#583]
When the "build in" RSS bot in the matrix room posts something that message will not appear in the relayed telegram group

### How to reproduce

1. Setup mautrix telegram with `bridge_notices.default:false` (which is the default configuration)
2. Relay a room to a telegram group
3. setup the RSS bot in the Matrix room
4. change the configuration to `bridge_notices.default:true` (`!tg config default` shows that the change was successful)
5. Subscribe to an RSS feed using the bot and wait until something new is posted in that feed

### What I expect

The message from the RSS bot that notifies about a new post in the RSS feed should show up in Telegram

### What actually happens

The message from the RSS bot appears in Matrix, but not in Telegram

### Additional Info

- Additionally issuing `!tg config set bridge_notices.default true` does not help either
- Using t2bot.io works, but using my selfhosted bot does not
- There also is t2bot.io's Discord bridge in the matrix room and it's messages get bridged over to telegram

### System info

- Homeserver and bot running via docker-compose
- Docker version 20.10.2, build 2291f61
- docker-compose version 1.27.4, build 40524192
- Synapse version 1.26.0
- mautrix-telegram 0.9.0+dev.db41c8d8

**Comment by maeries** [comment]
To try to work around the issue I created a new Matrix room and bridged that to the same telegram group as before, but the bot still doesn't bridge messages of the RSS bot

**Comment by penn5** [comment]
On telegram a bot can't see messages from a bot.

**Comment by maeries** [comment]
Sry that I didn't make it more clear, but the RSS bot I'm speaking of is a Matrix bot, not a telegram bot. So that can't be the issue here.

**Comment by JuniorJPDJ** [comment]
duplicate of https://github.com/tulir/mautrix-telegram/issues/573

**Comment by maeries** [comment]
> duplicate of #573

I don't think so, as in your screenshot I can see the message from the RSS bot appear in Telegram

**Comment by JuniorJPDJ** [comment]
Can you show screenshots of your behaviour?

**Comment by maeries** [comment]
![screenshot](https://user-images.githubusercontent.com/30597531/108111609-34034680-7095-11eb-9812-6f17b83a1c2b.jpg)
On the left, in Element, you can see a message from the RSS bot which is missing on the right side in Telegram

**Comment by JuniorJPDJ** [comment]
Can you post event json?

**Comment by maeries** [comment]
```
{
  "content": {
    "body": "Podcast: TEST ( https://podcast.xxx.xx/2021/02/15/test/ )",
    "format": "org.matrix.custom.html",
    "formatted_body": "<strong>Podcast</strong>:<br><a href=\"https://podcast.xxx.xx/2021/02/15/test/\"><strong>TEST</strong></a>",
    "msgtype": "m.notice"
  },
  "origin_server_ts": 1613415000515,
  "sender": "@_neb_rssbot_=40marius=3amatrix.xxx.xx:matrix.org",
  "type": "m.room.message",
  "unsigned": {
    "age": 102355526
  },
  "event_id": "$LRiJ1Vh7F6MqyrG4M6LGpvbi0bzUDr7-nlIaJ0veGmI",
  "room_id": "!ucWcbpAjWIngMuXtbb:matrix.xxx.xx"
}
```

I hope that's what the event json is

**Comment by JuniorJPDJ** [comment]
It's probably not bridged as it's `msgtype: m.notice`.
Could you check if you have enabled `bridge_notices` in your config?

**Comment by maeries** [comment]
I've set
```
    bridge_notices:
        # Whether or not Matrix bot messages (type m.notice) should be bridged.
        default: true
        # List of user IDs for whom the previous flag is flipped.
        # e.g. if bridge_notices.default is false, notices from other users will not be bridged, but
        #      notices from users listed here will be bridged.
        exceptions:
```
Though, I didn't set it that way right from the beginning so maybe the problem is that the bridge initially set `bridge_notices:default:false` and now fails to change it afterwards. On the other hand `!tg config defaults` also says `bridge_notices: default: true`

**Comment by JuniorJPDJ** [comment]
Check what is it set on group, as afaik it can be overwritten in per-group basics.

**Comment by maeries** [comment]
i issued `!tg config set bridge_notices.default true` some time ago in the room already and `!tg config view` shows 
```
1 bridge_notices:
2     default: true
3
```
So I guess the per-room config is also set to true?

**Comment by JuniorJPDJ** [comment]
That's true.
